.eleventy.js

```diff
@@ -153,6 +153,22 @@ module.exports = function (config) {
     return memoize(collection.getAll());
   });
 
+  // Filters through all collection items and finds content that has
+  // CSS_ORIGIN set to 'next'. This allows shortcodes to determine if we
+  // are in a design system context or a legacy context
+  config.addCollection('designSystemGlobals', (collection) => {
+    global.__designSystemPaths = [
```

Comment:
I think you should actually keep this as a `Set`, and use `.has()` in the helper.